graduated/helm/step.yaml

```diff
@@ -203,16 +203,26 @@ spec:
                 "type": "boolean",
                 "description": "Use debian based cfstep-helm image",
                 "default": false
+            },
+            "image": {
+                "type": "string",
+                "description": "Override the default image used for the step"
+                "default": "quay.io/codefreshplugins/cfstep-helm"
+            },
+            "debian_image": {
```

Comment:
There is no need to add a debian_image as a separate argument because it is hidden under condition and the user can't set both images at the same step. Also the user can use an image argument to set a custom debian image.